manticore/__main__.py

```diff
@@ -36,6 +36,8 @@ def positive(value):
                         help=argparse.SUPPRESS)
     parser.add_argument('--env', type=str, nargs=1, default=[], action='append',
                         help='Specify symbolic environment variable VARNAME=++++++')
+    parser.add_argument('--function', type=str, default=None,
```

Comment:
I feel that `--entry` (and `entry` in general for variable naming, etc) is a more descriptive name than `function`. It more accurately represents what we're selecting, and is less confusing imho. However, since the current code dispatches based on name, I'd suggest adding `--entry` that takes an address and `--entry-symbol` which takes a string and resolves it correctly.

Reply:
Ok, doing entry-symbol first